Assets/MixedRealityToolkit/Providers/Hands/HandJointService.cs

```diff
@@ -33,9 +33,12 @@ public class HandJointService : BaseInputDeviceManager, IMixedRealityHandJointSe
         /// <inheritdoc />
         public override void LateUpdate()
         {
+            if (InputSystem == null) { return; }
```

Comment:
>InputSystem [](start = 16, length = 11)

What's the case where InputSystem is null? It looks like in BaseInputDeviceManager it will set this in the constructor, so this one doesn't appear to be one that tries to get things later.

Reply:
This protects against the input device manager being passed a null instance. Perhaps this should be an assert instead.

Reply:
removed the check. it was inconsistent compared to other providers.

Reply:
Thanks - this was a case where it felt like it makes no sense for a hand joint service to start up without an input system backing it - it's a fundamental dependency of the class, and we should fail earlier (i.e. in the constructor) if we're not given required dependencies.